tendrl/node_agent/tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pytest
+import re
+
+
+@pytest.fixture
+def definitions_path():
```

Comment:
I would suggest to:

* name this fixture something like `yaml_file_path`
* fixture to either contain list of all included yaml files in tendrl-node-agent or find all such  files on the fly
* make this a parametrized fixture, so that test case would be parametrized over all yaml files this fixture knows about